sysdeps/linux/generic/sysdeps.cpp

```diff
@@ -839,6 +839,28 @@ int sys_setpriority(int which, id_t who, int prio) {
 	return 0;
 }
 
+int sys_nice(int increment, int *new_nice) {
+	int current;
+	if (int e = sys_getpriority(0, 0, &current); e)
```

Comment:
It would be nice (pun unintended) to comment what the actual constant is supposed to be. I don't think that breaking ABI for this right now would be beneficial, as I have a few more fixes queued up in my ABI checker PR that I'll probably post early next week.